### ISSUE TITLE
datastax/lunastreaming 2.9.x official release

### ISSUE THREAD

**Issue by Kristof86** [datastax/pulsar#33]
I was wondering if it is already known when an official datastax/lunastreaming version will be released for pulsar 2.9.x.

We are currently in need of a debezium connector for mssql and that one was introduced in pulsar version 2.9. 

I've already tried the 2.9 debezium connector on the latest lunastreaming release but it results in NoSuchMethod errors when installing a debezium source. I also tried to take the debezium mssql connector from the 2.9 and rebuilt it against all dependencies of the 2.8. Well, the connector works but the 2.8 is based on debezium 1.0 (the 2.9 on debezium 1.7) and 1.0 dates back to 2019 so I'm not really confident in using that...

An official 2.9 release would really help me out. 

Thanks!

Kristof

**Comment by nicoloboschi** [comment]
@Kristof86 yesterday we added the mssql connector. You can try to use it by manually building it 
[The branch is this one https://github.com/datastax/pulsar/tree/2.8.3_ds](https://github.com/datastax/pulsar/tree/2.8.3_ds/pulsar-io/debezium/mssql)

**Comment by Kristof86** [comment]
@nicoloboschi 
We seem to experience some problems building the project. It cannot seem to find the following dependency.
`
    <bookkeeper.version>4.14.4.1.0.0</bookkeeper.version>`

Could you help me out?

**Comment by nicoloboschi** [comment]
it should be fixed now. Can you retry ?

**Comment by Kristof86** [comment]
I still get the following error
```
Could not resolve dependencies for project com.datastax.oss:pulsar:pom:2.8.3.1.0.4-SNAPSHOT: Could not find artifact com.datastax.oss:bookkeeper-server:jar:tests:4.14.4.1.0.0
```

**Comment by nicoloboschi** [comment]
did you pull the latest changes? that jar is here
[bookkeeper-server-4.14.4.1.0.0-tests.jar](https://repo.datastax.com/artifactory/datastax-public-releases-local/com/datastax/oss/bookkeeper-server/4.14.4.1.0.0/bookkeeper-server-4.14.4.1.0.0-tests.jar)

In the latest commit I added that repository to the Maven configuration: https://github.com/datastax/pulsar/commit/07abcb9090f6e367b8e376447b7d327857b9d44f. It should be able to find the dependency jar

**Comment by Kristof86** [comment]
Hello

Not sure what went wrong yesterday, I also had some conflicts with my own Nexus it seems.

We've rebuilt the mssql nar and it works! It also resolved some problems that we had with the custom .nar that I built on debezium 1.0. 

So thanks alot!

**Comment by nicoloboschi** [comment]
@Kristof86 we released Luna Streaming 2.8.0.1.1.26 which contains the MsSQL connector.
https://github.com/datastax/pulsar/releases/tag/ls280_1.1.26
